### ISSUE TITLE
Enable setting a default configuration for split pane views

### ISSUE THREAD

**Issue by mmgj** [sanity-io/sanity#1664]
When configuring a split pane view in structure builder it would be nice to be able to configure a specific combo of views as the default for a given document type. I.e. A document of type "blogPost" would always initially show the editor in the first pane and a web preview in the second.

**Comment by johnnypea** [comment]
Is there a recommended way to show/view two or panes by default when opening the document? Thanks.

**Comment by ricokahler** [comment]
If anyone happens to be stumbling upon this now, you can get default views by implementing [`getDefaultDocumentNode`](https://www.sanity.io/docs/structure-builder-reference#97e44ce262c9). This API isn't perfect so we'll leave this issue open for a future update 😄

**Comment by lauri865** [comment]
@ricokahler, how would one us `getDefaultDocumentNode` for that?

**Comment by fvieira** [comment]
I also feel this need, or maybe a way to make the split view pane open a different view from the one that was open to avoid having to click twice (once to open the split pane and again to change this view to the web preview).

**Comment by miked-known** [comment]
Yes, I'm also trying to do this as well -- would be a nice UX enhancement if this was possible in some way.

**Comment by miklschmidt** [comment]
Just leaving my +1 :)

**Comment by AbeCole** [comment]
Another +1

**Comment by kindrift-vt** [comment]
Has there been any mention of this in v3?

**Comment by dbateman** [comment]
+1

**Comment by Lkleven** [comment]
+1

**Comment by harrygreen** [comment]
+1

**Comment by pedrobonamin** [comment]
Hello everyone, thank you for your suggestions!
This will be supported from version `5.9.0` using the `defaultPanes` property.

```ts
export const defaultDocumentNode: DefaultDocumentNodeResolver = (S, {schemaType}) => {
  if (schemaType === 'article') {
    return S.document()
      .views([
        S.view.form().id('editor'),
        S.view.component(LivePreview).id('preview').title('Preview'),
        S.view.component(JSONView).id('json').title('JSON')
      ])
      .defaultPanes(['editor', 'preview'])  // Form + Preview side-by-side
  }
  return S.document()
}
```
